packages/runtime/container-runtime/src/test/gc/garbageCollection.spec.ts

```diff
@@ -330,6 +332,29 @@ describe("Garbage Collection Tests", () => {
 		});
 	});
 
+	it("Private Autorecovery API", () => {
```

Comment:
This is good, can you write an e2e test that repros the bug you saw in telemetry? I can't really predict the behavior of this PR at a glance